gcp_variant_transforms/testing/data/sharding_configs/integration_without_residual.yaml

```diff
@@ -28,3 +28,8 @@
      regions:
        - "chr1:200,000,000-999,999,999"
      total_base_pairs: 249,240,615
+-  output_table:
+     table_name_suffix: "empty"
+     regions:
+       - "does not match to any reference_name value"
```

Comment:
same here

Reply:
Modified to make it more clear I meant empty shard.